app/views/upload.scala.html

```diff
@@ -113,24 +113,29 @@ <h2 class="govuk-error-summary__title logged-out-title" id="logged-out-error-tit
     <div class="govuk-grid-column-two-thirds">
         @progressIndicator(Messages("upload.progress"))
         <h1 class="govuk-heading-l">Uploading records</h1>
-        <p class="govuk-body">Do not close your browser window while your files are being uploaded. This could take a few minutes.</p>
-        <div id="upload-progress-error" class="govuk-error-summary upload-error" hidden aria-labelledby="error-summary-title"
-            role="alert" tabindex="-1" data-module="govuk-error-summary">
-            <div id="upload-progress-error">
+        <div id="upload-progress-error" hidden>
+            <div class="govuk-error-summary upload-error" aria-labelledby="error-summary-title"
+                role="alert" tabindex="-1" data-module="govuk-error-summary">
                 <h2 class="govuk-error-summary__title" id="error-summary-title">
                     There is a problem
                 </h2>
                 <div class="govuk-error-summary__body">
                     <p>Some or all of your files failed to upload.</p>
-                    <p class="upload-progress-error-timeout__message" hidden>Your upload has timed out, click <a class="govuk-link" href="@routes.SeriesDetailsController.seriesDetails()">series</a> to start again.</p>
-                    <p class="upload-progress-error-authentication__message" hidden>You have been logged out, click <a class="govuk-link" href="@routes.SeriesDetailsController.seriesDetails()">series</a> to start again.</p>
-                    <p class="upload-progress-error-general__message" hidden>Something has gone wrong, click <a class="govuk-link" href="@routes.SeriesDetailsController.seriesDetails()">series</a> to start again</p>
+                    <p class="upload-progress-error-timeout__message" hidden>Your upload has timed out. Click 'Return to start' to begin a new transfer.</p>
+                    <p class="upload-progress-error-authentication__message" hidden>You have been signed out. Click 'Return to start' to begin a new transfer.</p>
+                    <p class="upload-progress-error-general__message" hidden>Click 'Return to start' to begin a new transfer.</p>
                 </div>
             </div>
+            <a class="govuk-button" href="@routes.DashboardController.dashboard()" role="button" draggable="false" data-module="govuk-button">
+            Return to start
+            </a>
         </div>
-        <div>
-            <span id="upload-status-screen-reader"><label for="upload-records-progress-bar" class="govuk-label progress-label"></label></span>
-            <progress class="progress-display" value="0" max="100"></progress>
+        <div id="progress-bar-and-message">
```

Comment:
Looking at this in the context of the existing elements, it's a bit counter-intuitive for the `<div>` on line 112 to be called `progress-bar` when this one is called `progress-bar-and-message`. They sound like they're nested in the wrong order.

Maybe the outer one should be called something more generic like `upload-progress`?

It would also be good to find a name for this one that doesn't list the elements, because it will be out-of-date if we change what's inside, but I'm struggling to think of a good name to use instead. 🙂